src/views/Gantt.vue

```diff
@@ -0,0 +1,216 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div class="c-gantt">
+    <v-container
+      fluid
+      class="pa-2"
+    >
+      <!-- Filters -->
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          md="4"
+          class="pr-md-2 mb-2 mb-md-0"
+        >
+          <v-text-field
+            id="c-gantt-filter-job-name"
+            clearable
+            placeholder="Filter by job name"
+            v-model.trim="jobsFilter.name"
+            ref="filterNameInput"
```

Comment:
This is filtering by *task* name (e.g. foo, bar, baz) rather than job ID (i.e. the Cylc submission number e.g. 01, 02, 03).

Reply:
Done.
I'm aware of performance / size issues you mention. 
The next thing to implement is pagination. This was going to be done in this pull request, but it turned out to be more work than I envisaged, so it was cut.